unconvert.go

```diff
@@ -356,8 +341,8 @@ type step struct {
 }
 
 type visitor struct {
-	pkg   *loader.PackageInfo
-	file  *token.File
+	pkg   *packages.Package
+	file  *ast.File
```

Comment:
I think this PR can be a lot less invasive if you change `pkg`'s type to `*types.Info` (and then a subsequent PR to rename `pkg` to `info`), and leave `file` as `*token.File`.

Reply:
As a side benefit: if we stick to just `*types.Info`, then we can potentially make `computeEdits` conditionally use `go/loader` or `go/packages` based on a compile-time or run-time flag in case users have problems with `go/packages`.

Reply:
I'll take a look tomorrow to that approach.

Reply:
I changed `pkg *loader.PackageInfo` to `info *types.Info` at master, so you should be able to get rid of these changes now and focus just on the `go/loader` to `go/packages` change in `computeEdits`. Cheers.

Reply:
This change was much nicer thanks.